### ISSUE TITLE
Any Typescript definitions for this?

### ISSUE THREAD

**Issue by lexfrost** [pixijs/pixi-particles#36]

**Comment by andrewstart** [comment]
There is https://github.com/pixijs/pixi-typescript/blob/master/pixi-particles.d.ts which may be a little out of date.

**Comment by wewo** [comment]
@andrewstart , can you please explain how to use it? I manually added definitions od pixi-particles code into my original pixi.js d.ts file and IDE now recognizes types and everything, but the actual compiled JavaScript doesn't know abou pixi-particles module.

My question is, how to include it in the project?
I have `import * as PIXI from 'pixi.js';` and that works for base PIXI, but I don't know how to let it (webpack?) know that I also want to use pixi-particles library. What should I import and where? I know this is a noob question but hopefully it's trivial and you can help me. Thanks!

**Comment by andrewstart** [comment]
I haven't tried it myself, but it looks like the .d.ts file adds the definitions to the global PIXI export, so you should need to add `import 'pixi-particles';` after your PIXI import and the imported PIXI should have the particle stuff in it.

**Comment by wewo** [comment]
@andrewstart this kinda worked.

Now my imports are
`import 'pixi.js';`
`import 'pixi-particles';`

I couldn't merge two d.ts files as I did it at first, I had to create its own folder `pixi-particles` module in `typings` folder and also `index.d.ts` file for it. You were right that it's out of date thought, but only small changes were needed for generated emitter from PixiParticlesEditor to work.

So thank you very much!
